srcs/SetServer_utils.cpp

```diff
@@ -0,0 +1,10 @@
+#include "../includes/Server.hpp"
+
+
+void Server::shut_down_server(int i, int fd)
+{
+    close(fd);
+    this->pfds[i] = this->pfds[this->pollfd_count - 1];
```

Comment:
Is this the way to actually rearrange the vector?
[https://stackoverflow.com/questions/875103/how-do-i-erase-an-element-from-stdvector-by-index]
shouldn't it be like this?

Reply:
Hey that approach is also valid, with erase:

void Server::shut_down_server(int i, int fd) {
close(fd);
this->pfds.erase(this->pfds.begin() + i);
}

At the end you will remove the element at that index. What im doing there is just replacing the pdfs[i], element with the last element of the vector. In that way i removes the element in that index. But at the end you will get the same result, but i think is more understandable if you use erase, soo if you want we can change it !